modules/plugins/languages/nix.nix

```diff
@@ -74,6 +74,22 @@
         }
       '';
     };
+
+    nixd = {
+      package = pkgs.nixd;
+      internalFormatter = cfg.format.type == "nixpkgs-fmt";
+      lspConfig = ''
+        lspconfig.nixd.setup{
+          capabilities = capabilities,
```

Comment:
Missing on_attach?

Reply:
Sorry but do you mind elaborating? I'm a noob at this and simply copied the format for rnix-lsp and tested it out on my config to see if it worked as expected.

Reply:
```suggestion
          capabilities = capabilities,
          on_attach = default_on_attach,
```

Reply:
Added to rebased commit.